demo/src/main/java/org/jboss/examples/ticketmonster/service/SeatAllocationService.java

```diff
@@ -2,17 +2,17 @@
 
 import java.util.List;
 
-import javax.inject.Inject;
-import javax.persistence.EntityManager;
-import javax.persistence.LockModeType;
-import javax.persistence.NoResultException;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.LockModeType;
+import jakarta.persistence.NoResultException;
 
 import org.jboss.examples.ticketmonster.model.Performance;
 import org.jboss.examples.ticketmonster.model.Seat;
 import org.jboss.examples.ticketmonster.model.SeatAllocationException;
 import org.jboss.examples.ticketmonster.model.Section;
 import org.jboss.examples.ticketmonster.model.SectionAllocation;
-import org.jboss.examples.ticketmonster.service.AllocatedSeats;
```

Comment:
der import wird glaub ich noch benötigt. Zumindest gibt die allocateSeats(...) methode noch ein AllocatedSeats Objekt zurück.

Reply:
Eclipse hat gesagt, es wird nicht benötigt. Hat in diesem Fall auch recht, weil die Klasse AllocatedSeats im selben Package ist.